### DISCUSSION TITLE
Disable comment symbol in new line

### DISCUSSION THREAD

**Discussion by MannyFay** [numToStr/Comment.nvim#366] [Q&A] [answered]
How I can disable putting the comment symbol in a new line if I press enter?

**Comment by otakutyrant** [comment, marked as answer]
It may be related to `formatoptions`. Read `h: formatoptions`.

**Comment by MannyFay** [comment]
@otakutyrant thanks to your hint :)

The following option in my `options.lua` has solved the problem:
```lua
vim.opt.formatoptions:remove({ 'r', 'o', 'n', 'l' })
```
